py3status/modules/xrandr_toggle.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Toggle monitor output from the status bar using xrandr.
+
+Configuration parameters:
+    cache_timeout: how often we refresh usage in seconds (default: 10s)
+    distext: custom text or symbol for disconnected status
```

Comment:
> 80 characters limit 'legacy'.

79 and it's not legacy in any way.  just because you can have monitors with 200 chars doesn't mean you should use them.  Long lines are hard to scan.  newspapers have columns because of readability.  Also email destroys lines over 80 chars quite a lot and it's nice sometimes to email code.

Also I tend to have several terminals on a screen so actually 79 chars is often more than a line.  Then there are people with accessibility issues screen readers/ people with poor eyesight.  Hey sometimes I look at code on my phone.  And if I need to scroll right to read code here on github I cry inside.

With the indenting issue it is about writing better code, not complaining about the language.  If you are finding it hard then maybe you need to figure out why that's the case.  We learn some very bad habits in our lives.